src-java/stats-topology/stats-messaging/src/main/java/org/openkilda/messaging/info/stats/RemoveFlowPathInfo.java

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.stats;
+
+import org.openkilda.messaging.info.InfoData;
+import org.openkilda.messaging.payload.flow.PathNodePayload;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.Value;
+
+import java.util.List;
+
+/**
+ * Represents remove path info.
+ */
+@Value
+@EqualsAndHashCode(callSuper = false)
+@JsonSerialize
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class RemoveFlowPathInfo extends InfoData {
+    private static final long serialVersionUID = 1L;
+
+    @JsonProperty("flow_id")
+    String flowId;
+
+    @JsonProperty("cookie")
+    long cookie;
```

Comment:
We should not loose type info here. Let receiver make required format changes, not sender.

Reply:
Changed.